sdk/keyvault/azure-security-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/CertificateService.java

```diff
@@ -197,7 +196,7 @@
                                                    Context context);
 
     @Get("deletedcertificates/{certificate-name}")
-    @ExpectedResponses({200, 404})
+    @ExpectedResponses({200, 403, 404})
```

Comment:
we need to do this in beginCreateCertificate Polling operation service call, not here.